charts/kubeclarity/values.yaml

```diff
@@ -203,12 +219,46 @@ kubeclarity-postgresql:
   ## Setup database name and password
   existingSecret: kubeclarity-postgresql-secret
   postgresqlDatabase: kubeclarity
+  # secretKey: postgresql-password
 
-  ## Enable security context
+  serviceAccount:
+    enabled: true
+  persistence:
+    enabled: true
+    accessMode: ReadWriteOnce
+    size: 20Gi
+    storageClass:
+    annotations:
+      helm.sh/resource-policy: "keep"
+  replication:
+    enabled: false
+  service:
+    port: 5432
+  resources:
+    limits:
+      cpu: 2
+      memory: 2Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+  securityContext:
+    # Default is true for K8s. Enabled needs to false for OpenShift restricted SCC and true for anyuid SCC
+    enabled: true
+    # fsGroup specification below is not applied if enabled=false. enabled=false is the required setting for OpenShift "restricted SCC" to work successfully.
+    fsGroup: 1001
   containerSecurityContext:
+    # Default is true for K8s. Enabled needs to false for OpenShift restricted SCC and true for anyuid SCC
     enabled: true
+    # runAsUser specification below is not applied if enabled=false. enabled=false is the required setting for OpenShift "restricted SCC" to work successfully.
     runAsUser: 1001
-    runAsNonRoot: true
+  volumePermissions:
+    enabled: true
+    # if using restricted SCC set runAsUser: "auto" and if running under anyuid SCC - runAsUser needs to match the line above
+    securityContext:
+      runAsUser: 1001
+  shmVolume:
+    chmod:
+      enabled: false
```

Comment:
why false?

Reply:
From postgres chart values.yaml:
>  Init container Security Context
>  @param volumePermissions.securityContext.runAsUser User ID for the init container
>  Note: the chown of the data folder is done to securityContext.runAsUser
>  and not the below volumePermissions.securityContext.runAsUser
>  When runAsUser is set to special value "auto", init container will try to chwon the
>  data folder to autodetermined user&group, using commands: `id -u`:`id -G | cut -d" " -f2`
>  "auto" is especially useful for OpenShift which has scc with dynamic userids (and 0 is not allowed).
>  You may want to use this volumePermissions.securityContext.runAsUser="auto" in combination with
>  pod securityContext.enabled=false and shmVolume.chmod.enabled=false

Reply:
Should be `true` by default for alignment with K8s deployments. Code updated and description/comment added.